doc/lightning-sendcustommsg.7.md

```diff
@@ -16,8 +16,9 @@ top, not for direct use by end-users.
 
 The message must be a hex encoded well-formed message, including the 2-byte
 type prefix, but excluding the length prefix which will be added by the RPC
-method. The messages must not use even-numbered types, since these may require
-synchronous handling on the receiving side, and can cause the connection to be
+method. The messages must not use even-numbered types, unless using
+`--experimental-even-onion-messages`, since these may require synchronous
```

Comment:
You also want to modify `doc/lightningd-config.5.md`.

Reply:
done